app/models/feedback.rb

```diff
@@ -14,5 +14,5 @@ class Feedback
 
   attr_accessor :comment, :rating, :email
   validates :comment, presence: true
-  validates :rating, inclusion: { in: RATINGS.keys.map(&:to_s) }
+  validates :rating, inclusion: { in: RATINGS.keys.map(&:to_s), message: 'Please select a valid rating' }
```

Comment:
this should use a translation. create a file in `config/locales/en/models/feedback.yml` and add them there. look at the search and search_filter files for error examples

Reply:
using the yml file method will mean you do not need to put the message in here at all, i believe

Reply:
also, from a message content point of view, do not use `please`  -typically a radio option like this would just say `choose a rating` or similar

Reply:
👍 Thank you, I have added a message for invalid rating to a translation file, it is a nice solution.

Reply:
I have made the message "Choose a rating".